Dockerfiles/Dockerfile-ubuntu-22.04

```diff
@@ -16,7 +16,6 @@ RUN \
 
 # Get carta dependencies carta-casacore, fits2idia, and zfp from cartavis-team PPA and casacore-data from KERN
```

Comment:
The end of this comment should also be removed.

Reply:
Well spotted